### DISCUSSION TITLE
Domínio para o SRD

### DISCUSSION THREAD

**Discussion by pedroleone** [pedroleone/osr-srd#14] [General]
Pessoal, o que acham de comprar um domínio para o SRD? Poderíamos fazer uma Vakinha ou algo do tipo, com o pessoal do grupo de Whatsapp do OSE. @nerun você acha que teria adesão esse tipo de coisa?

**Comment by nerun** [comment]
Pelo que me lembre domínios não são caros. Em 2020 eu paguei R$364,00 pelo prazo de 10 anos do [gurpzine.com.br](https://www.gurpzine.com.br). É mais barato quando compra por um prazo longo. O preço unitário, por um ano é R$40,00 ([registro.br](https://registro.br/ajuda/pagamento-de-dominio/)). A economia é pequena, mas não tem a amolação de renovar anualmente. Tem outros preços e prazos também. Tem que ver essa tabela do link do registor.br.

No caso do OSE, tem esse domínio disponível:
* oldschoolessentials.com.br

Esses já foram adquiridos:
* oldschool.com.br
* ose.com.br
* essentials.com.br

De qualquer forma, acho que é um pouco cedo pra isso. Por hora o [ose-srd.netlify.app](https://ose-srd.netlify.app/) tá tranquilo.

**Reply by pedroleone** [reply]
Perfeito! Eu estou pensando em mudar do Netlify para o GitHub Pages, dessa forma você mesmo conseguiria fazer o deploy caso necessário. Aí pensei em já aproveitar para colocar o domínio, mas se você acha cedo (o que eu concordo) podemos deixar para o futuro. Valeu!
